pom.xml

```diff
@@ -229,6 +229,13 @@
       <version>3.1.0</version>
     </dependency>
     <dependency>
+      <groupId>redis.clients</groupId>
+      <artifactId>jedis</artifactId>
+      <version>2.9.0</version>
+      <type>jar</type>
+      <scope>compile</scope>
```

Comment:
`type` and `scope` can be left out since the default will do.

Reply:
👍  removed `type` and `scope`